components/layouts/columns/columns-two-wide-left.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="section--bg-white">
+  <div class="columns--two-left-wide section__inner">
+
+    <div class="column--left">
+      <picture src="https://via.placeholder.com/625x938" alt="ALT TEXT GOES HERE">
+        <source media="(min-width: 866px)" srcset="https://via.placeholder.com/625x938">
+        <source media="(min-width: 0px)" srcset="https://via.placeholder.com/640x360">
+        <img src="https://via.placeholder.com/625x938" alt="ALT TEXT GOES HERE">
+      </picture>
+    </div>
+
+    <div class="column--right">
+
+      <h3 class="basic-content__title">Flexible</h3>
+      <div class="basic-content__text">
+        <p>And we mean really flexible. Why be contained to a traditional career path, the four walls of an office, or
+          the typical 9-5?</p>
+      </div>
+
+      <h3 class="basic-content__title">Empowering</h3>
+      <div class="basic-content__text">
+        <p>Your career. Your call. We give you the opportunities, tools, and guidance to create the career you want.</p>
+      </div>
+
+      <h3 class="basic-content__title">Inclusive</h3>
+      <div class="basic-content__text">
+        <p>We’re all in when it comes to diversity. We’re constantly working to build a more inclusive environment where
+          everyone can contribute, grow, thrive—and express themselves.</p>
+        <p>Learn more about <a href="#">Equity &amp; Inclusion</a> at Vitamin T.</p>
+      </div>
+
+      <h3 class="basic-content__title">Equal</h3>
+      <div class="basic-content__text">
+        <p>Whether you’re a full-time staffer or a part-time freelancer, we’ve got your back with great benefits and HR
+          support.</p>
+      </div>
```

Comment:
Let's make a modification here just to make sure things present right. Since this column will be one entire piece of content, we won't have classes on the headings or wrappers around paragraphs. Just mark it up as bare `<h3>`s and `<p>`s, and confirm that the presentation tracks correctly. If it doesn't, let's talk about how to get it correct without using BEM classes and without leaking into other patterns.

Reply:
Thank you for letting me know.